src/net_permissions.cpp

```diff
@@ -21,9 +21,10 @@ const std::vector<std::string> NET_PERMISSIONS_DOC{
 namespace {
 
 // Parse the following format: "perm1,perm2@xxxxxx"
-bool TryParsePermissionFlags(const std::string& str, NetPermissionFlags& output, size_t& readen, bilingual_str& error)
+static bool TryParsePermissionFlags(const std::string& str, NetPermissionFlags& output, ConnectionDirection* output_connection_direction, size_t& readen, bilingual_str& error)
 {
     NetPermissionFlags flags = NetPermissionFlags::None;
+    ConnectionDirection connection_direction = ConnectionDirection::None;
```

Comment:
Does this mean that `in` and `out` are now required in conf options? If you defaulted to `::In` here would that maintain backwards compatibility?

Reply:
Update: oh I see the default now on line 75 below... why not set that here when you initialize the variable?

Reply:
It's cleaner when/if the default is changed to both.

Reply:
> why not set that here when you initialize the variable?

I think the reason is the `|=`operator.